src/button/_button.scss

```diff
@@ -14,292 +14,186 @@
  * limitations under the License.
  */
 
-@import "../variables";
-@import "../mixins";
-
-// The button component. Defaults to a flat button.
 .mdl-button {
-  background: transparent;
+  min-height: 36px;
   border: none;
-  border-radius: $button-border-radius;
-  color: $button-secondary-color;
+  min-width: 88px;
   position: relative;
-  height: $button-height;
-  margin: 0;
-  min-width: $button-min-width;
-  padding: 0 $button-padding;
-  display: inline-block;
-  @include typo-button();
+  color: rgb(0, 0, 0);
+  background: transparent;
+  border-radius: 2px;
+  /*
+  Add 6 top and bottom padding.
+  Provides 12 total for a 48px high touch area.
+  */
+  padding: 6px 16px;
+  font-family: $performance-font;
+  font-weight: 500;
+  text-transform: uppercase;
+  display: inline-flex;
+  font-size: 0.875rem;
```

Comment:
Let's do the comment for any rem calcs. That way we aren't relying on more preprocessing tooling. If we do got to another processor, we want to minimize what we do so everything can eventually "just ship", so that's the way to go.